debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -2157,7 +2157,239 @@ access_driver_type
+      ( ',' ET_COL ':' ( quoted_string | '[' quoted_string ( ',' quoted_string )* ']' ))? RIGHT_CURLY_PAREN
+    ;
+
+col_comment
+    : quoted_string
+    ;
+
+overflow_element
+    : LEFT_CURLY_PAREN ET_ACTION ':' ( ET_TRUNCATE | ET_ERROR )
+      ( ',' ET_COL ':' ( quoted_string | '[' quoted_string ( ',' quoted_string )* ']' ))? RIGHT_CURLY_PAREN
+    ;
+
+et_data_type
+    : ( et_primitive_type
+        ARRAY '<' et_data_type '>'
+        MAP '<' et_primitive_type ',' et_data_type '>'
+        STRUCT '<' field_spec et_data_type ( COMMENT ) (',' COMMENT )* '>'
+        UNIONTYPE '<' et_data_type ( ',' et_data_type ) '>' )
+    ;
+
+et_primitive_type
+    : TINYINT
+    | SMALLINT
+    | INT
+    | BIGINT
+    | BOOLEAN
+    | FLOAT
+    | DOUBLE
+    | STRING
+    | BINARY
+    | TIMESTAMP
+    | DECIMAL
+    ;
+
+et_location_specifier
+    : quoted_string
+    ;
+
+comments_oracle_loader
+    : RECORDS DELIMITED BY NEWLINE_
+    ;
+
+comments_oracle_datapump
+    : NOLOG
+    ;
+record_format_info
+    : ( RECORDS ( FIXED UNSIGNED_INTEGER
+                | VARIABLE UNSIGNED_INTEGER
+                | DELIMITED BY ( (DETECTED) NEWLINE_ | quoted_string )
+                | XMLTAG '('? quoted_string ( ',' quoted_string )* ')'? ))?
+      et_record_spec_options*?
+    ;
+
+et_record_spec_options
+    : CHARACTERSET quoted_string
+    | EXTERNAL VARIABLE DATA
+    | PREPROCESSOR et_directory_spec? et_file_spec
+    | DATA IS ( LITTLE | BIG ) ENDIAN
+    | BYTEORDERMARK ( CHECK | NOCHECK )
+    | STRING SIZES ARE IN ( BYTES | CHARACTERS )
+    | LOAD WHEN et_condition_spec
+    | et_output_files
+    | ( READSIZE '='? UNSIGNED_INTEGER | DISABLE_DIRECTORY_LINK_CHECK | DATE_CACHE UNSIGNED_INTEGER | SKIP_ UNSIGNED_INTEGER )
+    | IO_OPTIONS ( DIRECTIO | NODIRECTIO )
+    | ( DNFS_ENABLE | DNFS_DISABLE )
+    | DNFS_READBUFFERS UNSIGNED_INTEGER
+    ;
+
+et_directory_spec
+    : directory_name object_name? ':'
+    ;
+
+et_file_spec
+    : filename
+    ;
+
+et_condition_spec
+    : et_condition | et_condition_spec ( AND | OR ) et_condition_spec
+    ;
+
+et_condition
+    : ( field_spec |  ( '(' UNSIGNED_INTEGER ':' UNSIGNED_INTEGER ')' ))
+        relational_operator
+      ( quoted_string | HEX_STRING_LIT | BLANKS  )
+    ;
+
+
+et_output_files
+    : ( NOBADFILE |  BADFILE et_directory_spec? et_file_spec )
+    | ( NODISCARDFILE | DISCARDFILE et_directory_spec? et_file_spec )
+    | ( NOLOGFILE | LOGFILE et_directory_spec? et_file_spec )
+    ;
+
+field_definitions
+    : FIELDS '('?
+        ( IGNORE_CHARS_AFTER_EOR )?
+        ( CSV ( WITH EMBEDDED | WITHOUT EMBEDDED ))?
+        et_delim_spec?
+        et_trim_spec?
+        ( ALL FIELDS OVERRIDE THESE FIELDS )?
+        ( MISSING FIELD VALUES ARE NULL_)?
+        ( REJECT ROWS WITH ALL NULL_ FIELDS)?
+        ( DATE_FORMAT ( DATE | TIMESTAMP ) MASK quoted_string )?
+        ( NULLIF ( EQUALS_OP | NOT_EQUAL_OP ) ( quoted_string | HEX_STRING_LIT | BLANKS ) | NONULLIF )?
```

Comment:
I believe the `HEX_STRING_LIT` definition in the Lexer needs to be adjusted to account for `0x'<value>'` which is used in the supplied SQL files and the current variant doesn't match.  I'm thinking:
```
HEX_STRING_LIT: ('X'|'x'|'0X'|'0x') ('\'' [A-F0-9]* '\'')+;
```